### PR TITLE
fix vector-dimension and configProducer retry

### PR DESCRIPTION
**Pull request by Yuqi-Du**
**Checklist**
- [x] Changes manually tested
- [x] Automated Tests added/updated
- [x] Documentation added/updated
- [x] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

**Review by Hazel-Datastax — APPROVED** [review]

File: src/main/java/io/stargate/sgv2/jsonapi/service/embedding/configuration/EmbeddingProvidersConfigProducer.java [hunk 1]
```diff
@@ -26,10 +26,10 @@ public class EmbeddingProvidersConfigProducer {
   @ApplicationScoped
   @Startup
   @Retry(
-      maxRetries = 1,
+      maxRetries = 30,
```
Comment (tatu-at-datastax) [inline src/main/java/io/stargate/sgv2/jsonapi/service/embedding/configuration/EmbeddingProvidersConfigProducer.java:29]: Shouldn't this be in other PR, to avoid bundling unrelated changes?

Comment (tatu-at-datastax) [reply src/main/java/io/stargate/sgv2/jsonapi/service/embedding/configuration/EmbeddingProvidersConfigProducer.java:29]: Actually it's fine either way.

**Review by tatu-at-datastax — APPROVED** [review]

File: src/main/java/io/stargate/sgv2/jsonapi/service/embedding/configuration/EmbeddingProvidersConfigProducer.java (hunk 1 above)
Comment (Yuqi-Du) [reply src/main/java/io/stargate/sgv2/jsonapi/service/embedding/configuration/EmbeddingProvidersConfigProducer.java:29]: Yeah, I forget to change it back yesterday.

Comment (tatu-at-datastax) [reply src/main/java/io/stargate/sgv2/jsonapi/service/embedding/configuration/EmbeddingProvidersConfigProducer.java:29]: ah.
